full-moon/src/ast/mod.rs

```diff
@@ -2152,7 +2152,7 @@ impl<'a> Ast<'a> {
                         stmts: Vec::new(),
                         last_stmt: None,
                     },
-                    tokens,
+                    eof: tokens.pop().unwrap(),
```

Comment:
Can you change the `unwrap` to `expect` with a clear error message that there was no EOF?

Reply:
Following the example between these two instances, I've added a comment on why it should never panic. I could still change it to an `.expect` if you prefer.

Reply:
I don't think it'll ever panic in practice, but it's good to have the `expect` anyway.